src/QueryEditor.tsx

```diff
@@ -149,22 +154,35 @@ export class QueryEditor extends PureComponent<Props> {
     return (
       <div>
         {options.query.rawQuery && (
-          <InlineFieldRow>
-            <InlineField
-              label="Cassandra CQL Query"
-              labelWidth={30}
-              tooltip="Enter Cassandra CQL query. Also you can use $__timeFrom and $__timeTo variables, it will be replaced by chosen range"
-              grow
-            >
-              <TextArea
-                placeholder={'Enter a CQL query'}
-                onChange={this.onQueryTextChange}
-                onBlur={this.props.onRunQuery}
-                value={this.props.query.target}
-              />
-            </InlineField>
-            <Button icon="pen" variant="secondary" aria-label="Toggle editor mode" onClick={this.onChangeQueryType} />
-          </InlineFieldRow>
+          <>
+            <InlineFieldRow>
+              <InlineField
+                label="Cassandra CQL Query"
+                labelWidth={30}
+                tooltip="Enter Cassandra CQL query. Also you can use $__timeFrom and $__timeTo variables, it will be replaced by chosen range"
+                grow
+              >
+                <TextArea
+                  placeholder={'Enter a CQL query'}
+                  onChange={this.onQueryTextChange}
+                  onBlur={this.props.onRunQuery}
+                  value={this.props.query.target}
+                />
+              </InlineField>
+              <Button icon="pen" variant="secondary" aria-label="Toggle editor mode" onClick={this.onChangeQueryType} />
+            </InlineFieldRow>
+            <InlineFieldRow>
+              <InlineField label="Alias" labelWidth={8} tooltip="Series name override. Plain text or template using column names, e.g. `{{ column1 }}:{{ column2}}`">
+                <Input
+                    name="alias"
+                    value={this.props.query.alias || ''}
+                    onChange={this.onAliasChange}
+                    onBlur={this.props.onRunQuery}
+                    width={22}
```

Comment:
And remove this one, so the input field takes the all available row length

Reply:
I removed this parameter, but the field doesn't got bigger. [Screenshots ](https://github.com/HadesArchitect/GrafanaCassandraDatasource/pull/172#discussion_r1356646285) were made without explicit `width`. I guess `TextArea` and `Input` fields have different defaults.

Reply:
Mhm... Not sure. Let's make it at least 30 then, to match the label, or even 60?